synapse/rest/client/v2_alpha/account.py

```diff
@@ -599,6 +599,9 @@ def __init__(self, hs):
         return 200, {"threepids": threepids}
 
     async def on_POST(self, request):
+        if self.hs.config.disable_3pid_changes:
+            raise SynapseError(400, "3PID changes disabled on this server")
```

Comment:
```suggestion
            raise SynapseError(400, "3PID changes are disabled on this server")
```

(same with the 2 others below)